src/main/java/visualizations/LineChart.java

```diff
@@ -0,0 +1,47 @@
+package visualizations;
+
+import tech.tablesaw.api.Table;
+import tech.tablesaw.plotly.Plot;
+import tech.tablesaw.plotly.api.LinePlot;
+import tech.tablesaw.plotly.components.Figure;
+import tech.tablesaw.plotly.components.Layout;
+import tech.tablesaw.plotly.components.Line;
+import tech.tablesaw.plotly.traces.ScatterTrace;
+
+/**
+ * Class that plots basic Line Charts.
+ */
+
+public class LineChart {
+
+    /**
+     * Plots a line given the chart title,
+     * data set in Table format,
+     * and two numeric columns.
+     */
+    public static void plotLine(
+            String chartTitle, Table table, String x, String y) {
+
+        Plot.show(LinePlot.create(chartTitle, table, x, y));
+
+    }
+
+    /**
+     * Plots a line given the chart title,
+     * data set in Table format,
+     * two numeric columns and custom
+     * smoothing.
+     */
+
+    public static void plotSmoothLine(
```

Comment:
You can see why we should wrap the plotting options in this function. What happens when we want a different colour say? when another font is required? When the image should be saved instead of shown on the browser?